.github/workflows/twig-integration.yml

```diff
@@ -0,0 +1,34 @@
+name: Twig Check
+
+on: [push, pull_request]
+
+jobs:
+  twig_cs_check:
+    name: Twig Check
+    runs-on: ubuntu-latest
+
+    steps:
+      # Checkout the repository to access the code
+      - name: Checkout code
+        uses: actions/checkout@v4
+
+      # Set up PHP with Composer
+      - name: Set up PHP
+        uses: shivammathur/setup-php@v2
+        with:
+          php-version: '8.2'  # Use the appropriate PHP version for your project
```

Comment:
Is this the comments of an AI?

Reply:
No, it's a comment from the docs I wrote, but you're right, I can remove it. 👍